openslides_backend/models/models.py

```diff
@@ -239,6 +239,7 @@ class Meeting(Model):
     jitsi_domain = fields.CharField()
     jitsi_room_name = fields.CharField()
     jitsi_room_password = fields.CharField()
+    enable_chat = fields.BooleanField()
```

Comment:
Please add this field to meeting.update (group f, next to the jitsi fields)

Reply:
Done. Added field enable_chat to meeting.update in group f.